Makefile

```diff
@@ -192,12 +192,12 @@ e2e-image: docker-build
 
 # Pull all the images references in test/e2e/data/e2e_conf.yaml
 test-e2e-image-prerequisites:
-	docker pull gcr.io/k8s-staging-cluster-api/cluster-api-controller:v1.3.0
-	docker pull gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller:v1.3.0
-	docker pull gcr.io/k8s-staging-cluster-api/kubeadm-control-plane-controller:v1.3.0
-	docker pull quay.io/jetstack/cert-manager-cainjector:v1.8.2
-	docker pull quay.io/jetstack/cert-manager-webhook:v1.8.2
-	docker pull quay.io/jetstack/cert-manager-controller:v1.8.2
```

Comment:
These were all out of sync with the e2e_conf.yaml, meaning that we were pulling one version and then trying to load another version into the kind cluster. The result was basically that we pulled multiple images multiple times. Tests should be slightly faster when these are in sync.